internal/pkg/handler/updated-handler.go

```diff
@@ -211,7 +209,7 @@ func rollingUpgradeForStatefulSets(client kubernetes.Interface, r ResourceUpdate
 
 func updateContainers(containers []v1.Container, annotationValue string, sshData string, resourceType string) bool {
 	updated := false
-	envar := "STAKATER_" + convertToEnvVarName(annotationValue) + resourceType
+	envar := "STAKATER_" + helper.ConvertToEnvVarName(annotationValue) + resourceType
 	logrus.Infof("Generated environment variable: %s", envar)
```

Comment:
Please mention  the name of Deployment, Daemonsets or statefulset on which action is being performed